code/modules/projectiles/updated_projectiles/guns/pistols.dm

```diff
@@ -112,8 +112,8 @@
 	recoil_unwielded = CONFIG_GET(number/combat_define/min_recoil_value)
 
 /obj/item/weapon/gun/pistol/m1911/custom
-	name = "\improper M1911A1 Custom pistol"
-	desc = "A 20th century military firearm that received several modifications. It seems to have been lovingly taken care of and passed down the family. Lacks an auto magazine eject feature"
+	name = "\improper Colt Government Mk. IV Series 100"
```

Comment:
Lengthy name which is still a sinonymous of the previous one.

Reply:
That was a suggestion by someone else. I'll change it out.